neptune/experiments.py

```diff
@@ -61,9 +61,11 @@ class Experiment(object):
         Column sorting
     """
 
-    def __init__(self, client, leaderboard_entry):
+    def __init__(self, client, _id, internal_id, project_full_id):
```

Comment:
Why is there an underscore in `_id` argument name?

Reply:
`_id` is a built-in function in python: https://docs.python.org/3/library/functions.html#id